app/src/pages/inside/projectSettingsPageContainer/content/elements/fieldElement/fieldElement.jsx

```diff
@@ -41,7 +42,7 @@ export const FieldElement = (props) => {
   const getChildren = () =>
     withoutProvider ? children : <FieldProvider {...rest}>{children}</FieldProvider>;
   return (
-    <div className={cx('wrapper', className)} data-automation-id={dataAutomationId}>
+    <div className={cx('wrapper', className)} data-automation-id={dataAutomationId} title={title}>
```

Comment:
I suppose it would be better to extend `Checkbox` component with `title` property and use it directly for it.
We typically use tooltips and titles for interactive elements rather than their parent containers.

Reply:
Done.